mpqp/core/instruction/gates/native_gates.py

```diff
@@ -252,6 +255,9 @@ def to_other_language(
     def to_matrix(self) -> Matrix:
         return self.matrix
 
+    def to_canonical_matrix(self):
```

Comment:
return type missing, and why not putting self.matrix directly ?

Reply:
what do you mean by not putting self.matrix directly ?

Reply:
we created this "canonical matrix" thing to emphasize the difference, but maybe it is not needed indeed, we should have a quick discussion about this this afternoon

Reply:
Sorry I was not clear, I mean in `to_matrix` we return `self.matrix`, and then in `to_canonical_matrix` we call the method `to_matrix`, my question was why we don't just directly return also `self.matrix` in `to_canonical_matrix`, instead of calling a method in a method (it is not super heavy in this case, but if we can simply avoid, it is better I think)

Reply:
done